src/components/ItemForm/ItemForm.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+import './ItemForm.css';
+import { v4 as uuid} from 'uuid';
+import { postClosetData } from '../../apiCalls';
+
+const ItemForm = () => {
+  const [image, setImage] = useState("");
+  const [notes, setNotes] = useState("");
+  const [category, setCategory] = useState("");
+  const [error, setError] = useState({error:false, message: ""});
+  const [addSuccess, setAddSuccess] = useState(false);
+
+  const [newData, setNewData] = useState({
+    id: "",
+    image: "",
+    category:"",
+    notes: ""
+  });
+
+  const handleChange = (e) => {
+    setAddSuccess(false);
+    console.log(e.target.files);
+    setImage(URL.createObjectURL(e.target.files[0]));
+    console.log('file state', URL.createObjectURL(e.target.files[0]))
+}
+
+  const handleSelect = (e) => {
+    setAddSuccess(false);
+    setCategory(e.target.value);
+  }
+
+  const handleNotes = (e) => {
+    setAddSuccess(false);
+    setNotes(e.target.value);
+  }
+  const clearForm = () => {
+    setImage("")
+    setNotes("")
+    setCategory("")
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    if(image && category) {
+      setError({error: false, message: ""});
+      setAddSuccess(true);
```

Comment:
I think we should move this to a `.then` after the post function because it shows up even if the item was not successfully added.

Reply:
Okay i moved it- let me know if this is what you were thinking. I tested it and it works!